src/api/models.ts

```diff
@@ -26,10 +26,11 @@ export interface Timelog {
   projectName: string
   clientId: string
   isCurrentlyRunning: boolean
-  timeLogId: string
```

Comment:
This might be necessary here, I had issues with this and it was needed for some API. So you might need a separate model where the L is lowercase.